application/models/exercise.py

```diff
@@ -0,0 +1,23 @@
+from application import db
+from enum import Enum
+
+class SomaticCategory(Enum):
```

Comment:
Glad we got this figured out together @Chulstro @sienna-kopf!

Reply:
Love these enums. so cool y'all